### DISCUSSION TITLE
When shall this component support the carbon V11 so we can use `@carbon/react`

### DISCUSSION THREAD

**Discussion by xalc** [carbon-design-system/ibm-products#2108] [General] [answered]
Now this component have many peer dependences such as [carbon-components-react](https://www.npmjs.com/package/carbon-components-react)
but it  is not necessary for carbon V11

**Comment by elycheea** [comment, marked as answer]
@xalc We’re working on a release candidate for a v11 version of the package. We’re expecting the initial version to be available Q3.

**Reply by elycheea** [reply]
@xalc Our [pre-release version](https://github.com/carbon-design-system/ibm-cloud-cognitive/compare/@carbon/ibm-products@2.0.0-rc.0...@carbon/ibm-products@2.0.0-rc.1) is now available if you’d like to test it out.
